src/main/scala/uk/gov/hmrc/initprototype/HerokuReportTask.scala

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.initprototype
+
+import java.io.PrintWriter
+import java.io.File
+
+import scala.concurrent.{Await, ExecutionContext, Future}
+
+class HerokuReportTask(implicit herokuManager: HerokuManager, executionContext: ExecutionContext) {
+  import herokuManager._
+
+  def getAppReleaseInfo(appName: String): Future[String] = {
+    val releasesFuture  = getAppReleasesRecursive(appName)
+    val formationFuture = getAppFormation(appName)
+
+    for ((releases, _) <- releasesFuture; formationOption <- formationFuture) yield {
```

Comment:
Might be easier to read if these two statements for the for block are on separate lines.